### ISSUE TITLE
Icon Request: fa-door

### ISSUE THREAD

**Issue by vzorglub** [FortAwesome/Font-Awesome#11590]
for home automation
example:
fa-door-open
fa-door-ajar
fa-door-closed

A whole set for home automation would be great!
I am thinking amongst others:
fa-window-open, ajar, closed
light-bulb-on, off
radiator-on, off
thermostat - various states
hvac-heating, cooling, off, fan-only...
PIR-on, off
garagedoor-open, ajar, closed
fridge
freezer
living-room
bedroom
kitchen
conservatory
attic
aquarium
fire
hallway
landing
bathroom
toilet

Basically, a whole home automation set.
Thanks

**Comment by arunkapil** [comment]
Couldn't the 'thermometer' set of icons fulfill the 'thermostat' request?

**Comment by Siege36** [comment]
I'd also like to request a home automation icon pack.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Unfortunately, I cannot leave this request open because it doesn't meet our [Contributing Guidelines](https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#contributing-to-font-awesome).

Please make sure that your request follows our [new icon request template](https://github.com/FortAwesome/Font-Awesome/issues/new?title=Icon%20request:%20icon-name&template=icon-request.md)
